myuw/dao/class_website.py

```diff
@@ -19,9 +20,15 @@ def _fetch_url(method, url):
         logger.error("_get_html_from_url(%s)==>%s" % (url, ex))
         return None
 
-    pool = get_con_pool("%s://%s" % (p.scheme, p.netloc), socket_timeout=2)
-    response = get_live_url(
-        pool, method, p.hostname, url, {'ACCEPT': 'text/html'})
+    headers = {'ACCEPT': 'text/html'}
+    if p.netloc == 'localhost':
```

Comment:
I'd rather see this switch happen w/ settings, rather than hostname.  If not, the check you put in for security below - replacing '..' with '' should be specifically tested.